cocoa/events_darwin.go

```diff
@@ -46,6 +46,11 @@ func (w *Window) EventChan() (events <-chan interface{}) {
 	downKeys := make(map[string]bool)
 	ec := make(chan interface{})
 	go func(ec chan<- interface{}) {
+
+		var noX int = 1<<31 - 1
```

Comment:
Actually, I'm not in close to a linux computer.  I'll let someone else change the linux code if that is desired.